src/libraries/System.Private.CoreLib/src/System/Globalization/EastAsianLunisolarCalendar.cs

```diff
@@ -11,9 +11,8 @@ public abstract class EastAsianLunisolarCalendar : Calendar
         private const int nDaysPerMonth = 3;
 
         // # of days so far in the solar year
-        private static readonly int[] s_daysToMonth365 = { 0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334 };
-
-        private static readonly int[] s_daysToMonth366 = { 0, 31, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335 };
+        private static ReadOnlySpan<int> DaysToMonth365 => new int[] { 0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334 };
```

Comment:
There are 6 copies of this pattern throughout corelib. Might be a good opportunity for consolidation into a utility class.